95.md

```diff
@@ -0,0 +1,108 @@
+NIP-95
+======
+
+Relay File Storage
+------------------
+
+`draft` `optional`
+
+This NIP defines a way to store binary data on relays. It supports parallel uploads and resumable downloads.
+
+## Upload
+
+To upload a file, first client must convert its bytes to base64. It may do it in chunks made of multiples of 3 bytes or in one go.
```

Comment:
3 bytes?

Reply:
multiples of 3 cause base64 needs atleast 3 bytes to encode. it could be 255000 bytes per chunk for example

Reply:
Bad english, tried to improve text